src/classes/i18n.js

```diff
@@ -199,15 +209,82 @@ export default class i18n {
     }
 
     get_lang_files (lang, cb) {
+        const MAX_RETRY = this.config.get('download_retry');
 
         // append a random string, to avoid getting a response from cache
         const random_str = '&rand=' + ~~(Math.random() * 1000);
         const url = this.translations_url.replace(':lang', lang) + random_str;
-        const write_stream = fs.createWriteStream(this.config.get('locale_dir') + lang + '.json');
 
-        request(url)
-            .pipe(write_stream)
-            .on('finish', cb);
+        const translation_file_path = this.config.get('locale_dir') + lang + '.json';
+
+        let promise_chain = Promise.resolve();
+        if (process.env.REFRESH_TRANSLATIONS) {
```

Comment:
Would it be more consistent if we 
1. check for force reload and file exists and version matching.
2. Clean folder
3. Then download the expected 
matching version?

Reply:
> check for force reload and file exists and version matching.

Hmm, when you say "force reload", do you mean to download then check version? We already have that here. 

As for the `Clean folder`, I guess we can add that and remove `json` files.

Reply:
There's no need to check for version if we already download?

Reply:
Updated here. I've separate the version check and checking of the file https://github.com/anyTV/anytv-i18n/pull/10/commits/44c08ac0c0bd945156e9f74056c292f1d96df6b3